compiler/rustc_error_codes/src/error_codes/E0523.md

```diff
@@ -0,0 +1,15 @@
+Found two different crates that are not distinguished by differing metadata.
+
+Erroneous code example:
+```
+// FIXME
```

Comment:
Note:  this is failing the verify check on push because it does not include `E0523`.